recipes/sylph/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "0.0.2" %}
+
+package:
+  name: sylph
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/bluenote-1577/sylph/archive/v{{ version }}.tar.gz
+  sha256: 434eabc39fa9ab5249c6a449b0646384c3a3bc4cc9dc60e2ed1354e2b6f87fb6
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - rust >=1.61
+    - make
+    - cmake >=3.12
+  host:
+  run:
```

Comment:
Are you sure you have no runtime dependencies in your package?

Reply:
Other rust-only executables such as https://github.com/bioconda/bioconda-recipes/blob/9ee5c10ded10632f01b19dab8565e7deb12dd6a6/recipes/rust-mdbg/meta.yaml#L4 or https://github.com/bioconda/bioconda-recipes/blob/9ee5c10ded10632f01b19dab8565e7deb12dd6a6/recipes/skani/meta.yaml#L4  seem to work okay without other runtime dependencies.